python/lsst/daf/butler/core/config.py

```diff
@@ -458,17 +464,11 @@ def _processExplicitIncludes(self) -> None:
                         found = fileName
                     else:
                         for dir in searchPaths:
-                            if isinstance(dir, ResourcePath):
-                                specific = dir.join(fileName.path)
-                                # Remote resource check might be expensive
-                                if specific.exists():
-                                    found = specific
-                            else:
-                                log.warning(
-                                    "Do not understand search path entry '%s' of type %s",
-                                    dir,
-                                    type(dir).__name__,
-                                )
+                            specific = dir.join(fileName.path)
+                            # Remote resource check might be expensive
+                            if specific.exists():
+                                found = specific
+                                # TODO: do we need `break` here?
```

Comment:
@timj, I think that we might want a `break` here, don't we?

Reply:
Yes, looks like it should break out the loop once it has found something in the search path...

Reply:
OK, I'll add a `break` and re-run Jenkins.